ui/app/adapters/variable.js

```diff
@@ -0,0 +1,30 @@
+import ApplicationAdapter from './application';
+import { pluralize } from 'ember-inflector';
+import classic from 'ember-classic-decorator';
+
+@classic
```

Comment:
`question` `non-blocking`: Do we need the `classic` decorator here?

https://github.com/emberjs/ember-classic-decorator

Reply:
Good question, and it appears that we do need it here (and in our equivalent serializer). This keeps convention with our other adapters, at the very least. I think it would be a good idea to get away from @classic altogether soon, but not in this PR.